docs/advanced-transfer/experimental-features.mdx

```diff
@@ -29,8 +29,18 @@ The `experimental_features` parameter accepts an array of strings, where each st
 
 **Identifier:** "eureka"
 
+### Layerzero (`"layer_zero"`)
+
+**Description:**  
+Improves cross-chain capabilities to support more assets such as USDT.
+
+**Opt-out switch:**  
+Layerzero routing was auto-activated on **March 1, 2025**.
```

Comment:
None of these are auto activated so we should prob remove this auto-activated part